generic/tclXcoalesce.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * tclXcoalesce.c --
+ *
+ *  coalesce Tcl commands.
+ *-----------------------------------------------------------------------------
+ * Copyright 2017 - 2019 Karl Lehenbauer and Mark Diekhans.
+ *
+ * Permission to use, copy, modify, and distribute this software and its
+ * documentation for any purpose and without fee is hereby granted, provided
+ * that the above copyright notice appear in all copies.  Karl Lehenbauer and
+ * Mark Diekhans make no representations about the suitability of this
+ * software for any purpose.  It is provided "as is" without express or
+ * implied warranty.
+ *-----------------------------------------------------------------------------
+ */
+
+#include "tclExtdInt.h"
+
+
+/*-----------------------------------------------------------------------------
+ * TclX_CoalesceObjCmd --
+ *     Implements the TCL chmod command:
```

Comment:
Should say `Implements the Tcl coalesce command`

Reply:
Nice catch.  Thank you.